src/govuk/settings/_links.scss

```diff
@@ -4,43 +4,62 @@
 
 /// Enable new link styles
 ///
+/// If enabled, the link styles will change. Underlines will:
+///
+/// - be consistently thinner and a bit further away from the link text
+/// - have a clearer hover state, where the underline gets thicker to make the
+///   link stand out to users
+///
+/// These changes are an opt-in feature because you may:
+///
+/// - need to make changes to your service to introduce these new link styles
+///   consistently
+/// - want to avoid opting in if you have links within a multiple-column layout
+///   in your CSS - there is [a Chromium bug that affects such
+///   links](https://github.com/alphagov/govuk-frontend/issues/2204)
+///
+/// These new link styles will be enabled by default in a future major version
+/// of GOV.UK Frontend.
+///
 /// @type Boolean
 /// @access public
 
 $govuk-new-link-styles: false !default;
 
-/// Link underline thickness
+/// Thickness of link underlines
```

Comment:
See my comment about whether these summaries are needed up top but - would it be better if we described these consistently with a verb? For example the one above has 'Enable'. So could this - and similar ones - use 'Set'?

Reply:
@m-green I wondered about this too...

So, there are 2 files in this PR, each representing different sections of the Sass docs:

- src/govuk/helpers/_links.scss - this documents mixins
- src/govuk/settings/_links.scss - this documents variables

As I understand it, mixins make your code do something, for example make links use a certain colour.

Whereas variables are settings that you change to apply to all of GOV.UK Frontend. So, variables are almost more like switches or toggles. I think if you use a variable, you're effectively just setting something to 'on' or 'off,' like choosing to include or not include the new link styles.

((In the future,  as we revise this content further, we'll hopefully also do something to differentiate mixins from variables.)

I think the rationale here (correct me if I'm wrong, @36degrees) for not prefacing summaries with a verb is: users will know that they have to set/enable a variable. In the same way that you see the word 'Volume' above a stereo knob, but not instructional text on what to do.

It's true that the variable above this one has the summary "Enable new link styles." The reason we included the verb there is because just saying "New link styles" is the only variable we believe users are likely to customise.

If we wanted to be consistent with the verb-first approach of "Enable new link styles" throughout the other variables, we'd have content like:

> **$govuk-link-underline-thickness**
> Sets offset of link underlines from text baseline
> Set this variable to `false` to avoid setting an offset.

How does that compare with the verb-less version?

> **$govuk-link-underline-thickness**
> Offset of link underlines from text baseline
> Set this variable to `false` to avoid setting an offset.

Reply:
If we wanted to be consistent with the verb-first approach of "Enable new link styles" throughout the other variables, we'd have content like:

> **$govuk-link-underline-thickness**
> Sets offset of link underlines from text baseline
> Set this variable to `false` to avoid setting an offset.

Is that potentially a bit confusing? Because we're saying 'Sets' (as in, "This is what the variable does") and then 'Set' (as in "This is what you, the user, should do").

Reply:
@36degrees Just talked to @m-green and we're going to revisit this issue another day. For now, we'll leave the wording as you originally suggested.